default.nix

```
@@ -5,7 +5,7 @@ in {
   nubank = {
     dart = callPackage ./pkgs/dart {};
     flutter = callPackage ./pkgs/flutter {};
-    hover = callPackage ./pkgs/hover {
+    hover = callPackage ./pkgs/hover-nix {
```

We can renamed it to `hover` only if now works, right?
```suggestion
    hover = callPackage ./pkgs/hover {
```

---

Done, sorry I forgot to remove it.